web_widget_one2many_product_picker/README.rst

```diff
@@ -0,0 +1,209 @@
+==================================
+Web Widget One2Many Product Picker
+==================================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fweb-lightgray.png?logo=github
+    :target: https://github.com/OCA/web/tree/12.0/web_widget_one2many_product_picker
+    :alt: OCA/web
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/web-12-0/web-12-0-web_widget_one2many_product_picker
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/162/12.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+Adds the 'one2many_product_picker' friendly mobile widget to create one2many lines linked with product.product records.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+Create or edit a new view and use the new widget called 'one2many_product_picker'.
+
+Widget options:
+~~~~~~~~~~~~~~~
+
+* product_per_page > Integer -> Used to control the auto-load behaviour (scroll-pagination)
+* groups > Array of dictionaries -> Declare the groups
+
+    * name -> The group name
+    * string -> The text displayed
+    * domain -> Forced domain to use
+    * order -> The order
+
+        * name -> The field name to order
+        * asc -> Flag to use 'asc' order
+
+* currency_field > Model field used to format monetary values
+* field_map > Dictionary:
+
+    * product -> The field that represent the product_id
+    * name -> The field that represent a name
+    * product_uom -> The field that represent a product_uom
+    * product_uom_qty -> The field that represent a product_uom_qty
+    * price_unit -> The field that represent a price_unit
+    * parent_id -> The field that represent the main record
```

Comment:
This should be got automatically from the one2many, and there's no possibility of being changed. Why it should be indicated?